### PR TITLE
Api authentication

### PR DESCRIPTION
**Pull request by adambraimbridge**

### CODE REVIEW

**Comment by adambraimbridge** [comment]
Don't merge this until after this is merged: http://git.svc.ft.com/projects/NEXTPRIVATE/repos/config-vars/pull-requests/59/overview

**Comment by adambraimbridge** [comment]
http://git.svc.ft.com/projects/NEXTPRIVATE/repos/config-vars/pull-requests/59/overview is merged, and the BEACON_API_KEY environemtn variable is available at https://dashboard.heroku.com/orgs/financial-times/apps/ft-next-beacon-dashboard/settings

File: Makefile [hunk 1]
```diff
@@ -29,12 +30,18 @@ ifeq ($(S3O_PUBLIC_KEY),)
 	@echo "You need a S3O_PUBLIC_KEY to run this app, speak to someone from the Next team."
 	exit 1
 endif
+# The Beacon API Key is available at http://git.svc.ft.com/projects/NEXTPRIVATE/repos/config-vars/browse/models/production.json
+ifeq ($(BEACON_API_KEY),)
+	@echo "You need a BEACON_API_KEY to run this app, speak to someone from the Next team."
+	exit 1
+endif
 	export apikey=${API_KEY}; \
 	export BASIC_AUTH="user:123"; \
```
Comment (commuterjoy) [inline Makefile]: delete this line

**Comment by commuterjoy** [comment]
+1

**Pull request merged**